meta/main.yml

```diff
@@ -0,0 +1,21 @@
+---
+galaxy_info:
+  author: haslersn
+  description: Role for https://github.com/librespot-org/librespot
+  company: stuvus (https://stuvus.uni-stuttgart.de)
+
+  license: MIT
+
+  min_ansible_version: 2.4
+
+  platforms:
+    - name: Debian
```

Comment:
Which versions? Not all support e.g. systemd

Reply:
I don't have time to test it on all Debian versions, so I added the one I tested it on.